_episodes/interviews/2023-02-03-adam-chalmers.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Cloudflare with Adam Chalmers"
+date: 2023-02-03T16:54:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2023-02-03-adam-chalmers.mp3
+duration: "01:06:25"
+length: "63763435"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Adam Chalmers](https://maxwellflitton.com/), software engineer at [Cloudflare](https://www.cloudflare.com/) which is a global network of servers located around the world that help power and secure the internet.
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+- [@0:00] - Introduction to Cloudflare
+- [@3:57] - Cloudflare's edge computing
+- [@7:56] - Running Rust binary on Cloudflare
+- [@10:36] - Adams' background and work history
+- [@15:12] - How does Cloudflare provide web security?
+- [@32:30] - Cloudflare’s transition and rewriting from C to Rust
+- [@37:12] - Cloudflare’s data plane and control plane
```

Comment:
There's a mix of ’ and ' symbols used here. In general, straight single quotes `'` are preferred.